externs/sourcebuffer.js

```diff
@@ -13,3 +13,6 @@
 
 /** @type {string} */
 SourceBuffer.prototype.mode;
+
+/** @type {Function} */
+// SourceBuffer.prototype.changeType;
```

Comment:
Why?

Reply:
This was firing a compiler type mismatch error previously. Testing with and without.